pyproject.toml

```diff
@@ -31,9 +31,24 @@ scikit-learn = "*"
 scipy = "*"
 tqdm = "*"
 
+[tool.poetry.group.dev]
+optional = true
+
 [tool.poetry.group.dev.dependencies]
 flake8 = "*"
 pytest = "*"
+pre-commit = "*"
+black = "*"
+isort = "*"
+mypy = "*"
```

Comment:
Sort alphabetically. Maybe we should also use https://github.com/andrei-shabanski/poetry-plugin-sort for development?

Reply:
Done.